### ISSUE TITLE
PSGalleryModule Tests for saved modules depend on PSv5

### ISSUE THREAD

**Issue by RamblingCookieMonster** [RamblingCookieMonster/PSDepend#31]
If a user specifies a target that is a path, rather than  scope, we test for existing modules using `Get-Module \\Path\To\Module -ListAvailable`.  This syntax works in PSv5 and later, does not work in PSv2, 3, or 4.

If you really need this, a workaround would be to use the PSGalleryNuget alternative

Given that all gallery modules should have a psd1, and thus any modules that could be the same will have this, we could use the `*-Metadata` functions from the Configuration module.

Low on my list of priorities, but open to pull requests!

**Comment by RamblingCookieMonster** [comment]
Closing this out.  Backwards compatibility is low priority IMHO, but if anyone wants it, feel free to re-open!
